scripts/exregional_run_met_genensprod_or_ensemblestat.sh

```diff
@@ -387,33 +356,54 @@ settings="\
   'obtype': '${OBTYPE}'
   'accum_hh': '${ACCUM_HH:-}'
   'accum_no_pad': '${ACCUM_NO_PAD:-}'
-  'field_thresholds': '${FIELD_THRESHOLDS:-}'
+  'metplus_templates_dir': '${METPLUS_CONF:-}'
+  'input_field_group': '${VAR:-}'
+  'input_level_fcst': '${FCST_LEVEL:-}'
+  'input_thresh_fcst': '${FCST_THRESH:-}'
+  'vx_config_dict': ${vx_config_dict:-}
 "
 
 # Render the template to create a METplus configuration file
 tmpfile=$( $READLINK -f "$(mktemp ./met_plus_settings.XXXXXX.yaml)")
-cat > $tmpfile << EOF
-$settings
-EOF
-
-uw template render \
-  -i ${metplus_config_tmpl_fp} \
-  -o ${metplus_config_fp} \
-  -v \
-  --values-file "${tmpfile}"
+printf "%s" "$settings" > "$tmpfile"
```

Comment:
The `cat` command sometimes fails to write all the contents of `settings` into `tmpfile` for some ensemble members when there are a lot of ensemble members (many rocoto tasks launched at the same time).  This happens, for example, when running the WE2E test `MET_ensemble_verification_winter_wx`.  It is not clear why.  Switching from `cat` to `printf` seems to fix the problem.